src/niswitch/metadata/__init__.py

```diff
@@ -10,12 +10,14 @@
 from metadata.functions_addon import functions_enums
 from metadata.functions_addon import functions_params_types
 from metadata.functions_addon import functions_buffer_info
+from metadata.functions_addon import functions_is_error_handling
 from metadata.functions_addon import function_default_value
 
 helper.merge_dicts(functions, functions_codegen_method)
 helper.merge_dicts(functions, functions_enums)
 helper.merge_dicts(functions, functions_params_types)
 helper.merge_dicts(functions, functions_buffer_info)
+helper.merge_dicts(functions, functions_is_error_handling)
```

Comment:
This is an unrelated fix, isn't it?

Reply:
Sort of? We were not merging this from functions.addon. In order to test the fix to use error_message, this needs to be added

Reply:
ok